src/Components/IndexScene/ToolbarScene.tsx

```diff
@@ -75,7 +75,15 @@ export class ToolbarScene extends SceneObjectBase<ToolbarSceneState> {
 
     const renderPopover = () => {
       return (
-        <div className={styles.popover} role="dialog" aria-modal="true" aria-label="Query options">
+        // This is already keyboard accessible, and removing the onClick stopPropagation will break click interactions. Telling eslint to sit down.
+        // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
+        <div
+          className={styles.popover}
+          role="dialog"
+          aria-modal="true"
+          aria-label="Query options"
+          onClick={(evt) => evt.stopPropagation()}
+        >
```

Comment:
Apologies this is from my a11y updates! Fix is good was able to toggle this off locally
![Screenshot 2025-05-06 at 12 35 52 PM](https://github.com/user-attachments/assets/84e50084-1917-4649-9c2a-e9cfb4910b3a)